ui/packages/consul-ui/app/routing/route.js

```diff
@@ -22,11 +22,14 @@ export default class BaseRoute extends Route {
       .filter(item => item !== 'index')
       .join('.');
     const template = get(routes, `${routeName}._options.template`);
-    if(template) {
+    if (template) {
       this.templateName = template;
     }
     const queryParams = get(routes, `${routeName}._options.queryParams`);
-    if(queryParams && (this.routeName === 'dc.partitions.index' || this.routeName === 'oauth-provider-debug')) {
+    if (
+      queryParams &&
+      ['dc.partitions.index', 'dc.nspaces.index', 'oauth-provider-debug'].includes(this.routeName)
```

Comment:
Small QOL changes to the code like this really just warm my heart 😂

Reply:
😂 

Yeah agree, as soon as things get past like 1 or 2 of these it's time to make it more readable.

FYI: This entire piece of the conditional shouldn't be here really, but is due to "reasons" - I'm hoping it will be gone by the time we hit 1.11 for real, if not it can disappear post 1.11.